.github/workflows/cloud-deploy/cloud-deploy-worker.yml

```diff
@@ -0,0 +1,137 @@
+name: Deploy PROD Worker
+
+on:
+  workflow_call:
+
+jobs:
+
+  build_prod_image:
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    needs: test_worker
+    timeout-minutes: 80
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** Can we time out of this job faster? 80 could hang the merge queue for some time.‏
```suggestion
    timeout-minutes: 15
```

Reply:
There are 5 occurrences to consider in the PR.